api/src/db/models/legacy_mixin/synopsis_mixin.py

```diff
@@ -71,6 +72,7 @@ class TsynopsisHistMixin:
     ac_email_addr: Mapped[str | None]
     ac_email_desc: Mapped[str | None]
     agency_name: Mapped[str | None]
+    top_level_agency_name: Mapped[str | None]
```

Comment:
If we keep this in this file, it will break the ELT process as these columns both need to exist in the legacy system and exactly match the order of the legacy system.
```suggestion
```